source/administration/changelog.md

```diff
@@ -2,6 +2,210 @@
+- Server logs will now be written to the `mattermost.log` file located at the path specified in **System Console > Logging > [File Log Directory](https://docs.mattermost.com/administration/config-settings.html#file-log-directory)**. Please set the file path in the System Console or in the `gitlab.rb` file if you are using GitLab Mattermost.
+
+#### Removed and deprecated features
+- Backwards compatibility with the old CLI tool is removed in v3.8. See [documentation to learn more about the new CLI tool](https://docs.mattermost.com/administration/command-line-tools.html).
+- Deprecated APIv3 routes removed in v3.8:
+   - `GET` at `/channels/more` (replaced by /`channels/more/{offset}/{limit}`)
+   - `POST` at `/channels/update_last_viewed_at` (replaced by `/channels/view`)
+   - `POST` at `/channels/set_last_viewed_at` (replaced by `/channels/view`)
+   - `POST` at `/users/status/set_active_channel` (replaced by `/channels/view`)
+- All APIv3 endpoints to be removed in September/2017 release (replaced by APIv4 endpoints)
+
+For a list of past and upcoming deprecated features, [see our website](https://about.mattermost.com/deprecated-features/).
+
+#### config.json   
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json` or the System Console.
+
+**Changes to Team Edition and Enterprise Edition**:
+
+ - Under `EmailSettings` in `config.json`:
+   - Added `"SkipServerCertificateVerification": false` to skip verification of smtp server certificates.
+
+**Additional Changes to Enterprise Edition**:
+
+ - Under `TeamSettings` in `config.json`:
+   - Added `"RestrictPrivateChannelManageMembers": all` to set who can add and remove members from private groups.
+
+### Database Changes
+
+**Posts Table:**
+- Added `IsPinned` column
+
+### API Changes
+
+**New routes (APIv3):**
+- `GET` at `/channels/{channel_id}/pinned`
+  - Returns the pinned posts in a channel
+- `POST` at `/channels/{channel_id}/posts/{post_id}/pin`
+  - Pins a post to a channel
+- `POST` at `/channels/{channel_id}/posts/{post_id}/unpin`
+  - Unpins a post from a channel
+  
+**Removed routes (APIv3):**
+- `GET` at `/channels/more` (replaced by /`channels/more/{offset}/{limit}`)
+- `POST` at `/channels/update_last_viewed_at` (replaced by `/channels/view`)
+- `POST` at `/channels/set_last_viewed_at` (replaced by `/channels/view`)
+- `POST` at `/users/status/set_active_channel` (replaced by `/channels/view`)
+
+### Websocket Event Changes
+
+**Added:**
+- `added_to_team` that occurs when the current user is added to a team by another user
+
+**Modified**
+- Added a `seq` field to websocket events that increments with each event sent to the client
+
+### Known Issues
+
+- Users are unable to sign up or login using GitLab. This issue is currently and will be released with v3.8.1
```

Comment:
do we have a link to the ticket?

Might be worth moving near the top of the changelog (e.g. where we're saying 3.8.1 with a security fix is coming out)